src/libpriv/rpmostree-scripts.c

```diff
@@ -388,6 +389,9 @@ run_script_in_bwrap_container (int rootfs_fd,
   /* Scripts can see a /var with compat links like alternatives */
   rpmostree_bwrap_var_tmp_tmpfs (bwrap);
 
+  if (glnx_fstatat (rootfs_fd, "usr/lib/opt", &stbuf, AT_SYMLINK_NOFOLLOW, NULL) && S_ISDIR(stbuf.st_mode))
+    rpmostree_bwrap_append_bwrap_argv (bwrap, "--symlink", "usr/lib/opt", "/opt", NULL);
```

Comment:
If this works for Chrome, then that's fine! Though I can imagine we'll have to switch to symlinking each `/opt` subdir here too if scriptlets want to write to other dirs in `/opt`. Anyway, we can do that later!